scripts/jenkins/Jenkinsfile

```diff
@@ -773,24 +817,50 @@ def xunitUpload() {
     ])
 }
 
+/* Create a new Docker Image Description
+ *
+ * @param name Name of the image, will be extended with registry, a common
+ *             prefix and a tag
+ * @param tagFun Closure describing how the tag should be formatted
```

Comment:
idFun? Why is this needed? How to use it? What are its arguments?

Reply:
how the image id should be set. (you want different ways depending on if it is a build image or for exmaple a homepage contaienr). pretty sure it is documented somewhere